airflow-core/src/airflow/api_fastapi/core_api/datamodels/plugins.py

```diff
@@ -65,20 +65,22 @@ class AppBuilderMenuItemResponse(BaseModel):
     model_config = ConfigDict(extra="allow")
 
     name: str
-    href: str | None = None
+    href: str
     category: str | None = None
 
 
-class IFrameViewsResponse(BaseModel):
+class ExternalViewResponse(BaseModel):
     """Serializer for IFrame Plugin responses."""
 
     model_config = ConfigDict(extra="allow")
 
     name: str
-    src: str
+    href: str
     icon: str | None = None
+    dark_mode_icon: str | None = None
```

Comment:
Reading multiple times I feel a "dark mode" should rather be a suffic for the property, would bring the benefit that a sorting lists light and dark mode together
```suggestion
    icon_dark_mode: str | None = None
```

Reply:
Good point, updated!